### PR TITLE
code refactoring

### PR DESCRIPTION
**Pull request by bhargavraviya**
Flutter Upgrade from scratch

- change code syntax and refactoring according new flutter version 

please check it and merge @YC

### CODE REVIEW

File: lib/config/routes.dart [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class AppRoutes {
+  static const String main = '';
```
Comment (YC) [inline lib/config/routes.dart]: The usage of `AppRoutes.main` in subsequent variables is a bit strange. 
Can you please change it to e.g. `static const String add = '/add';`

File: lib/pages/add.dart [hunk 2]
```diff
@@ -62,7 +62,6 @@ class _AddPageState extends State<AddPage> {
         Navigator.pop(context);
       });
     } catch (e) {
-      // TODO: Fix exception handling
```
Comment (YC) [inline lib/pages/add.dart]: This comment is still unresolved. 
Please add it back.

File: lib/pages/android/list_item.dart [hunk 3]
```diff
@@ -15,7 +15,7 @@ class HomeListItem extends TotpListItemBase {
 class _TOTPListItemState extends State<HomeListItem>
     with SingleTickerProviderStateMixin {
   // Dimension of progress indicator
-  static const double _PROGRESS_INDICATOR_DIMENSION = 25;
+  static const double progressIndicatorDimension = 25;
```
Comment (YC) [inline lib/pages/android/list_item.dart]: The variable should still be private

File: lib/pages/cupertino/home.dart [hunk 4]
```diff
@@ -78,7 +80,6 @@ class CupertinoHomePage extends StatelessWidget {
                 if (result == null) {
                   return;
                 }
-                // TODO: Transition to new type
```
Comment (YC) [inline lib/pages/cupertino/home.dart]: This TODO is not resolved. 
Please add it back.

File: lib/pages/cupertino/list_item.dart [hunk 5]
```diff
@@ -19,7 +19,7 @@ class HomeListItem extends TotpListItemBase {
 class _TOTPListItemState extends State<HomeListItem>
     with SingleTickerProviderStateMixin {
   // Dimension of progress indicator
-  static const double _PROGRESS_INDICATOR_DIMENSION = 25;
+  static const double progressIndicatorDimension = 25;
```
Comment (YC) [inline lib/pages/cupertino/list_item.dart]: The variable should still be private

File: lib/pages/settings.dart [hunk 6]
```diff
@@ -2,14 +2,16 @@ import 'package:flutter/widgets.dart';
...
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
-import 'package:another_authenticator/ui/adaptive.dart'
-    show AppScaffold, isPlatformAndroid;
-import 'package:another_authenticator/helper/url.dart' show launchURL;
+import '../ui/adaptive.dart' show AppScaffold, isPlatformAndroid;
+import '../helper/url.dart' show launchURL;
 import 'package:package_info_plus/package_info_plus.dart' show PackageInfo;
-import '../l10n/constants.dart' as Constants;
+import '../config/routes.dart';
+import '../l10n/constants.dart' as constants;
```
Comment (YC) [inline lib/pages/settings.dart]: Should probably turn this into a class and show Constants instead.

File: lib/pages/android/home.dart [hunk 7]
```diff
@@ -61,11 +64,11 @@ class AndroidHomePage extends StatelessWidget {
                 onTap: () async {
                   Navigator.pop(context);
 
-                  var result = await Navigator.pushNamed(context, "/add/scan");
+                  var result =
+                      await Navigator.pushNamed(context, AppRoutes.addScan);
                   if (result == null) {
                     return;
                   }
-                  // TODO: Transition to new type
```
Comment (YC) [inline lib/pages/android/home.dart]: This is not yet resolved. 
Please add comment back.

**Review by YC — CHANGES_REQUESTED** [review]

File: lib/config/routes.dart (hunk 1 above)
Comment (bhargavraviya) [reply lib/config/routes.dart]: remove static const String main = ''; line please check it

**Review by YC — APPROVED** [review]

**Comment by YC** [comment]
Thanks!

**Pull request merged**